doc/riverctl.1.scd

```diff
@@ -347,8 +347,20 @@ matches everything while _\*\*_ and the empty string are invalid.
 
 ## CONFIGURATION
 
-*attach-mode* *top*|*bottom*
-	Configure where new views should attach to the view stack.
+*attach-mode* *top*|*bottom*|*aside <N>*
+	Alias to default-attach-mode.
```

Comment:
I'd prefer a mention in the description of `default-attach-mode` such as "attach-mode may be used as an alias for this command for backwards compatibility"